src/portals/Screencopy.cpp

```diff
@@ -128,10 +128,16 @@ static void wlrOnBufferDone(void* data, zwlr_screencopy_frame_v1* frame) {
         PSESSION->sharingData.frameCallback = nullptr;
         Debug::log(LOG, "[screencopy/pipewire] Out of buffers");
         PSESSION->sharingData.status = FRAME_NONE;
+        if (PSESSION->sharingData.copyRetries++ < 6) {
```

Comment:
if 3 is too low, we can do 10. You can make that 10 a macro (or constexpr) at the top of the file

Reply:
Sure

Reply:
Changed it now, I don't use c++ very often so hopefully this is fine